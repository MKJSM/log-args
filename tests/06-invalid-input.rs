use log_args::params;

#[derive(Debug)]
struct User {
    id: u32,
    name: String,
}

// This should fail because `fields` is misspelled
#[params(field(user.id))]
fn process(user: User) {}

fn main() {}
