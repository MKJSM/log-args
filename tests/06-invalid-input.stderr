error: Unknown attribute, expected `fields` or `custom`
  --> tests/06-invalid-input.rs:10:10
   |
10 | #[params(field(user.id))]
   |          ^^^^^
